### PR TITLE
Fint drosjeloyve

### PR DESCRIPTION
**Pull request by erlingjahr**

### CODE REVIEW

File: charts/fint-drosjeloyve/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+name: fint-drosjeloyve
+description: A Helm chart for Kubernetes
```
Comment (fsjovatsen) [inline charts/fint-drosjeloyve/Chart.yaml]: Add a more descriptive description

File: charts/fint-drosjeloyve/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name}}
```
Comment (fsjovatsen) [inline charts/fint-drosjeloyve/templates/deployment.yaml]: use `fint-drosjeloyve.name` helper

File: charts/fint-drosjeloyve/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+              cpu: "1000m"
+            requests:
+              memory: "500Mi"
+              cpu: "500m"
+          ports:
+            - containerPort: 8080
+          readinessProbe:
+            httpGet:
+              path: /actuator/health/readiness
+              port: 8080
```
Comment (fsjovatsen) [inline charts/fint-drosjeloyve/templates/deployment.yaml]: move this to a value

**Review by fsjovatsen — CHANGES_REQUESTED** [review]
This service also needs the following services:
* fint-altinn-ebevis-service
* fint-altinn-download-queue-service
* fint-drosjeloyve-admin

File: charts/fint-drosjeloyve/templates/deployment.yaml (hunk 2 above)
Comment (erlingjahr) [reply charts/fint-drosjeloyve/templates/deployment.yaml]: Endra

File: charts/fint-drosjeloyve/templates/deployment.yaml (hunk 3 above)
Comment (erlingjahr) [reply charts/fint-drosjeloyve/templates/deployment.yaml]: Flytta til values

**Review by mmedalen — COMMENTED** [review]
I would use several different value files instead of putting all the municipalities in the same value file.

**Pull request merged**